pom.xml

```diff
@@ -62,7 +62,7 @@
         <!-- environment-dependent versions -->
         <zookeeper.version>3.4.5</zookeeper.version>
         <thrift.version>0.9.1</thrift.version>
-        <hadoop.version>2.2.0</hadoop.version>
+        <hadoop.version>2.5.2</hadoop.version>
```

Comment:
may not matter but we might want to build against 2.5.0 but suggest 2.5.2 for deploys. I would assume though that all 2.5.x are semi compat.